trade-finance-manager-ui/component-tests/case/facility.component-test.js

```diff
@@ -1,34 +0,0 @@
-const { pageRenderer } = require('../pageRenderer');
-
-const page = '../templates/case/facility/facility.njk';
-const render = pageRenderer(page);
-
-describe(page, () => {
```

Comment:
These tests are duplicated of the more thorough and correctly files tests in the /facility folder